Makefile

```diff
@@ -89,7 +89,7 @@ reset: \
 .PHONY: reset
 
 logs: ## display app logs (follow mode)
-	@$(COMPOSE) logs -f app
+	@$(COMPOSE) logs --tail="20" -f app
```

Comment:
20 is very small, an exception trace can be longer than that. You can safely increase it to 50 or 100.